src/main/java/edu/ucla/library/bucketeer/verticles/FinalizeJobVerticle.java

```diff
@@ -31,12 +49,28 @@
 
     private JsonObject myConfig;
 
+    private String myIiifUrl;
```

Comment:
Nitpicky/aesthetic bit: since IIIF and URL are both acronyms and generally capitalized, shouldn't this variable be named myIIIFURL?

Reply:
AFAIK, we don't really have a convention for this. A git-grep for "Iiif" and "Url" on this project yields some hits for variable/method names that had already existed, so that's why I went with this style. When I do a git-grep for "IIIF", the only variables that I get hits for are ones where the name is constant-style (i.e., all-caps, snake case). A git-grep for "URL" does show some non-constant variables, so ... I dunno. We don't really have a convention. I kind of think `myIIIFURL` looks somewhat like a constant, but I'm not that opinionated on this.

Reply:
My unspoken (obviously) convention on this is `IiifThing` and `thingOfIIIF` (all caps only when it's the last part of the string). I've done it both ways in the past, following different groups of people who've done it differently as their conventions, but have settled on the current pattern as the most readable (Constants are more readable with all caps because there my unspoken convention has been all caps with underscores separating the words). I think looking at the Services codebase you'll find this unspoken convention applied, but possibly not completely consistently. I'm not married to this if we want to present a case for something else, let's talk about it tomorrow in our zoom room(?)

Reply:
For context, I think I made the switch to lower case when using Vert.x's <code>HttpClient</code> (and related classes <code>HttpRequest</code>, etc.) so much, but even they aren't consistent, for instance: <code>HTTPRequestValidationHandler</code>.